From ce8803d2ec3a4045ccdc07cd8eed82dcfff637f4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=D0=BD=D0=B0=D0=B1?= <nabijaczleweli@nabijaczleweli.xyz>
Date: Tue, 11 Apr 2023 15:41:01 +0200
Subject: [PATCH] Bump ttywriteraw() limit to BUFSIZ
X-Mutt-PGP: OS

There doesn't appear to be a realistic reason to go any smaller than
this.
---
 st.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/st.c b/st.c
index 6a9df62..31c4f84 100644
--- a/st.c
+++ b/st.c
@@ -885,7 +885,7 @@ ttywriteraw(const char *s, size_t n)
 {
 	fd_set wfd, rfd;
 	ssize_t r;
-	size_t lim = 256;
+	size_t lim = BUFSIZ;
 
 	/*
 	 * Remember that we are using a pty, which might be a modem line.
-- 
2.30.2

